ksqldb-functional-tests/src/test/java/io/confluent/ksql/test/parser/SqlTestReaderTest.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.test.parser;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.instanceOf;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThrows;
+
+import io.confluent.ksql.parser.ParsingException;
+import io.confluent.ksql.parser.tree.AssertValues;
+import io.confluent.ksql.test.parser.TestDirective.Type;
+import org.junit.Test;
+
+@SuppressWarnings({"unchecked", "rawtypes"})
+public class SqlTestReaderTest {
```

Comment:
What happens if statements aren't separated by new lines?  Either directives or SQL statements?

I'm guessing for directives the second directive would just be in the text of the first, which is probably fine.  But can/does it support multiple SQL statements on a line, or indeed one statement spread across lines.  Tests needed :D

Reply:
> I'm guessing for directives the second directive would just be in the text of the first, which is probably fine.

yes

> But can/does it support multiple SQL statements on a line, or indeed one statement spread across lines.

It does support this, it uses our standard parser 😂 

>  Tests needed :D

Incoming